pkg/util/taints/taints.go

```diff
@@ -192,7 +192,6 @@ func deleteTaints(taintsToRemove []v1.Taint, newTaints *[]v1.Taint) ([]error, bo
 	allErrs := []error{}
 	var removed bool
 	for _, taintToRemove := range taintsToRemove {
-		removed = false
```

Comment:
hm, this demonstrates that this function's return bool doesn't make any sense, it reports a fact about the *last* thing rather than a fact about *all* the things.